modules/directives/src/main/scala/scala/build/errors/WrongSourcePathError.scala

```diff
@@ -0,0 +1,8 @@
+package scala.build.errors
+
+class WrongSourcePathError(exceptionMessage: String, cause: Throwable)
+    extends BuildException(
+      message = s"""The file path argument in the using directives at could not be found!
+                   |$exceptionMessage""".stripMargin,
```

Comment:
NIT Just in case, to be safer, I wouldn't call `stripMargin` on `exceptionMessage` (in case it has some `|` at the beginning of some lines)
```suggestion
                   |""".stripMargin + exceptionMessage,
```